Include workflow for ui lib unittests

kimwnasptd opened the pull request:
As part of setting up our CI/CD #5482 we want to extend the prow config to also run the unittests of the shared ui as a presubmit job.

/cc @PatrickXYS

PatrickXYS reviewed (approved):
/lgtm
/approve

PatrickXYS commented:
@kimwnasptd How about adding us into root folder's OWNERS to grant permission.

Otherwise, every time you change prow_config.yaml, you'll need to ask root OWNERS' permission

kimwnasptd commented:
@PatrickXYS you have a point. I'll make a PR to add us in the rook OWNERS file, so that we can move continue our CI/CD effort

kimwnasptd commented:
@PatrickXYS I added another commit here that removes the GCP workflow. The tests no seem to pass, although I can't find any relative logs from the underlying workflow.

Is it possible to see the logs from each Argo workflow task under the `Build Log` section in Prow's site? [ after clicking `Details` on the right of the bot ]

kimwnasptd commented:
@PatrickXYS I think we are set for this PR but feel free to add any more comments/thoughts you might have.

@Bobgy this is the PR that fixes the broken tests in `kubeflow/kubeflow`. As mentioned above we
* removed the older GCP workflow
* added another one that tests some common frontend code we have

So once we merge this feel free to also move forward with your work and remove `kubeflow/kubeflow` from the older infra https://github.com/kubeflow/kubeflow/issues/5482#issuecomment-782573046. So could you approve the PR in order to move forward?

PatrickXYS commented on prow_config.yaml:
```diff
@@ -1,15 +1,19 @@
+    job_types:
+      - presubmit
+    include_dirs:
+      - components/crud-web-apps/common/frontend/kubeflow-common-lib/*
```
The `include_dirs` will contain the directory that you wish to kick off the workflow.

But apprently in this PR, you haven't made any change to `components/crud-web-apps/common/frontend/kubeflow-common-lib/*`

Then there's no workflow

PatrickXYS commented:
> Is it possible to see the logs from each Argo workflow task under the Build Log section in Prow's site?

In short, it's by design, because those Argo workflow will flood the Build Log to say 10,000+ lines of logs, you don't want to check so.

The reason you haven't see any logs in Argo Workflow is because this PR doesn't kick off any workflow.

Bobgy reviewed (approved):
/lgtm
/approve
